tests/pipeline/test_engine.py

```diff
@@ -1502,7 +1503,7 @@ def dispatcher(c):
         )
 
 
-class ChunkedPipelineTestCase(zf.WithUSEquityPricingPipelineEngine,
+class ChunkedPipelineTestCase(zf.WithSeededRandomPipelineEngine,
```

Comment:
This test didn't actually care about using `EquityPricing` specifically; it just used the default values from the fixture and asserted that we got the same results running with or without chunks.

I switched to using `WithSeededRandomPipelineEngine` both to make that more clear, and to be get access to a string-dtype column.